content/en/account_management/rbac/_index.md

```diff
@@ -41,15 +41,13 @@ Datadog Read Only Role
 
 ## Custom roles
 
-<div class="alert alert-warning">
-Creating and modifying custom roles is an opt-in Enterprise feature. <a href="/help">Contact Datadog support</a> to get it enabled for your account.
-</div>
+Manage your custom roles through the Datadog application, the [Datadog Role API][6], or SAML directly. Find below how to create, update, delete a role. See the [Datadog Role Permissions][7] documentation for more information about available permissions. Only users with the Access Management permission can create or edit roles in Datadog.
 
-Manage your custom roles through the Datadog application, the [Datadog Role API][6], or SAML directly. Find below how to create, update, delete a role. See the [Datadog Role permissions][7] documentation for more information about available permissions. Only users with the Access Management permission can create or edit roles in Datadog.
+### Enable custom roles
 
-### Create a custom role
+TODO: Include description and screenshot showing how to enable custom roles through the datadog UI.
 
-You can create custom roles with:
```

Comment:
Removed this because this line was stylistically inconsistent with the other two sections, `Update` and `Delete`, and didn't provide any additional substance.